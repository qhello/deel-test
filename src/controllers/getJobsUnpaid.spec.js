const request = require("supertest");
const app = require("../app");

describe("getJobsUnpaid", () => {
  it("should return jobs", async () => {
    const res = await request(app).get("/jobs/unpaid").set("profile_id", 2);

    expect(res.statusCode).toEqual(200);
    expect(res.body).toMatchSnapshot();
  });
});
