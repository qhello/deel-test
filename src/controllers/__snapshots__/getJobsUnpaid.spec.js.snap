// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getJobsUnpaid should return jobs 1`] = `
[
  {
    "Contract": {
      "ClientId": 2,
      "ContractorId": 6,
      "createdAt": "2022-09-27T13:42:21.030Z",
      "id": 3,
      "status": "in_progress",
      "terms": "bla bla bla",
      "updatedAt": "2022-09-27T13:42:21.030Z",
    },
    "ContractId": 3,
    "createdAt": "2022-09-27T13:42:21.030Z",
    "description": "work",
    "id": 3,
    "paid": null,
    "paymentDate": null,
    "price": 202,
    "updatedAt": "2022-09-27T13:42:21.030Z",
  },
  {
    "Contract": {
      "ClientId": 2,
      "ContractorId": 7,
      "createdAt": "2022-09-27T13:42:21.030Z",
      "id": 4,
      "status": "in_progress",
      "terms": "bla bla bla",
      "updatedAt": "2022-09-27T13:42:21.030Z",
    },
    "ContractId": 4,
    "createdAt": "2022-09-27T13:42:21.030Z",
    "description": "work",
    "id": 4,
    "paid": null,
    "paymentDate": null,
    "price": 200,
    "updatedAt": "2022-09-27T13:42:21.030Z",
  },
]
`;
